packages/core/src/Stores/client-store.js

```diff
@@ -1644,11 +1645,12 @@ export default class ClientStore extends BaseStore {
                 BinarySocketGeneral.authorizeAccount(authorize_response);
 
                 // Client comes back from oauth and logs in
-                RudderStack.identifyEvent(user_id, {
+                Analytics.identifyEvent(user_id, {
```

Comment:
identify event happens on `setCoreAnalyticsData` in 1.3.1 and on `setAttributes` in 1.3.2 which we push with you soon